pkg/pool/reader.go

```diff
@@ -40,6 +42,19 @@ type ExclusionRange struct {
 	EndIP   string `json:"endIP"`
 }
 
+// Route contains a destination CIDR to be added as static route via gateway
+type Route struct {
+	Dst string `json:"dst"`
+}
+
+// RouteSlice is a type alias for a slice of Route
+type RouteSlice []Route
```

Comment:
nit: Do we need this type and functions below? It looks like it is not used.

Reply:
done